docs/reference/announcements-release-notes/880/880-announcements.md

```diff
@@ -342,3 +342,40 @@ Assignees list removed from response.
 These endpoints are superseded by [usage metrics endpoint](../../../apis-tools/orchestration-cluster-api-rest/specifications/get-usage-metrics.api.mdx).
 
 Operate and Tasklist usage metrics endpoints are **scheduled for removal in the 8.10 release**.
+
+### Process Instance Tags <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span><span class="badge badge--medium" title="This feature affects APIs">API</span> {#process-instance-tags}
+
+Camunda 8.8 adds optional, immutable **tags** for process instances you can set at creation via the Orchestration Cluster API or SDK. Tags provide lightweight, structured metadata for routing, correlation, prioritization and more without inspecting full variable payloads.
+
+**Highlights**
+
+- Added to `POST /v2/process-instances` (creation) and included in process instance and job responses.
+- Forwarded to every job created for the instance at element activation (immutable snapshot).
+- Exported with process instance and job entities (analytics / pipelines) starting in 8.8.
+- Exact-match AND filtering semantics when querying process instances by tags (instance must contain all supplied tags; may include more).
+- API-only in 8.8 (not visible in Operate or Tasklist UIs yet).
+
+**Constraints**
+
+| Limit / Rule | Value |
+|--------------|-------|
+| Max tags per instance | 10 unique |
+| Length | 1–100 characters |
+| Regex | `^[A-Za-z][A-Za-z0-9_\-:.]{0,99}$` (must start with a letter) |
+| Case | Case-sensitive (stored & matched exactly) |
+| Mutability | Immutable after creation |
+
+Invalid creation requests (too many, or malformed tags) are rejected (4xx) so invalid tags never propagate.
+
+**Recommended usage**
+
+Use `key:value` or `key` style labels (e.g. `businessKey:1234`, `priority:high`, `region:emea`, `customerId:7890` , `test`) for:
+
+- Correlation with external or internal systems.
+- Fast worker-side branching / prioritization prior to loading large variable sets.
+- Analytics segmentation & classification.
+
+Avoid secrets or PII—tags propagate broadly (jobs, exports).
```

Comment:
💭 Again, let's try to avoid duplicated content that is this specific. Linking to the definition should be fine here.